.github/CODEOWNERS

```diff
@@ -0,0 +1,98 @@
+# Numba's codeowners file is dual purpose, it:
+#
+# 1. Provides information to github about who should be requested to review a PR
+# 2. Provides users/contributors/czars general information about who to contact
+#    first about various parts of the code base. A lot of concepts in Numba are
+#    necessarily spread throughout the code base, consequently some of the
+#    "code ownership"/first contact is concept based opposed to file/directory
+#    based.
+#
+# ------------------------------------------------------------------------------
+# Information for github
+# ------------------------------------------------------------------------------
+# These people are the default "owners" for everything in the repo unless a
+# later match is made, they will automatically be requested to review PRs.
+* @sklam @stuartarchibald @esc
+
+# Owners of specific parts of the code, will be requested to review if a PR
+# touches code in the matched pattern
+/numba/cuda @gmarkall
+/numba/parfors @DrTodd13
+/numba/stencils @DrTodd13
```

Comment:
CC @DrTodd13 this should request your review for any parfors/stencil related PR.